modules/administration/pages/reports.adoc

```diff
@@ -0,0 +1,63 @@
+[[reports]]
+= Generate Reports
+
+The [command]``spacewalk-report`` command is used to produce a variety of reports.
+These reports can be helpful for taking inventory of your entitlements, subscribed systems, users, and organizations.
```

Comment:
```suggestion
These reports can be helpful for taking inventory of your subscribed systems, users, and organizations.
```

(That functionality has been removed since 3.0)